_posts/2018-02-01-snowplow-r100-epidaurus-released-with-pii-pseudonymization-support/index.md

```diff
@@ -0,0 +1,232 @@
+
+Concretely, the Snowplow operator is able to configure any and all of the fields whose values they wish to have hashed by Snowplow. Through hashing all the PII fields found within Snowplow events, you can minimize the risk of identification of a data subject - an important step towards meeting your obligations as data handlers.
+
+<h2 id="pii-enrichment">2. PII Enrichment</h2>
+
+This Snowplow release introduces the *PII Enrichment*, which provides capabilities for Snowplow operators to better protect the privacy rights of data subjects. The obligations of handlers of Personally Identifiable Information (PII) data under GDPR have been outlined on the [EU GDPR website][gdpr-web].
+
+This initial release of the PII Enrichment provides a way to *pseudonymize* fields within Snowplow enriched events. You can configure the  enrichment to pseudonymize any of the following datapoints:
+
+1. Any of the "first-class" fields which are part of the [Canonical event model][canonical-event-model], are scalar fields containing a single string and have been identified as being potentially sensitive
+2. Any of the properties within the JSON instance of a Snowplow self-describing event or context (wherever that context originated). You simply specify the Iglu schema to target and a [JSON Path][json-path] to identify the property or properties within to pseudonomize
+
+In addition, you must specify the "strategy" that will be used in the pseudonymization. Currently the available strategies involve *hashing* the PII, using one of the following algorithms:
+
+* `MD2`, the 128-bit algorithm [MD2][md2] (not-recommended due to performance reasons see [RFC6149][rfc6149])
+* `MD5`, the 128-bit algorithm [MD5][md5]
+* `SHA-1`, the 160-bit algorithm [SHA-1][sha-1]
+* `SHA-256`, 256-bit variant of the [SHA-2][sha-2] algorithm
+* `SHA-384`, 384-bit variant of the [SHA-2][sha-2] algorithm
+* `SHA-512`, 512-bit variant of the [SHA-2][sha-2] algorithm
+
+There is [a new Iglu schema][igc-schema] that specifies the configuration format for the PII Enrichment.
+
+Further capabilities for the PII Enrichment, including the ability to reverse pseudonymization in a controlled way, are planned for the [second phase of this PII Enrichment][r10x-pii2].
+
+<h2 id="pii-configure">3. Pseudonymizing your Snowplow events</h2>
+
+<h3>Before you start</h3>
+
+This brief tutorial assumes that you have gone through the [upgrading](#upgrading) section below, [deploying the latest version of Stream Enrich][setup-stream-enrich] and upgrading your Redshift table definition as necessary.
+
+<h3>Configuring the PII Enrichment</h3>
+
+Like all Snowplow enrichments, the PII Enrichment is configured using a JSON document which conforms to a JSON Schema, the [pii_enrichment_config][pii-config-schema], available in Iglu Central.
+
+Here is an example configuration:
```

Comment:
It can also be found here: https://github.com/snowplow/snowplow/blob/release/gdpr-1/3-enrich/config/enrichments/pii_enrichment_config.json

Reply:
Sure - we need to update this file (I will explain in HipChat)